app/templates/src/static/css/brand-palette.less

```diff
@@ -3,28 +3,67 @@
    Color variables
    ========================================================================== */
 
-// Official Palette as of October 29, 2013 (Brand 2.0)
+// Official Palette current as of April 9, 2015.
```

Comment:
Since we're editing this line, let's sentence case it ("Official palette").

Also, if you want to introduce the word "current", I think it should start on a new line:

```less
// Official palette
// Current as of April 9, 2015
```